ql/src/test/queries/clientpositive/lateral_view_cbo.q

```diff
@@ -0,0 +1,8 @@
+--! qt:dataset:src
```

Comment:
nit: We don't really need data for the plans here so we can drop the usage of `src`. A very simple DDL (`CREATE TABLE simple_t (col STRING)`) suffices and would make the test self-contained.

Reply:
Done